server/helpers/index.js

```diff
@@ -1,7 +1,9 @@
 import createToken from './createToken';
 import responseMessage from './responseMessage';
+import identify from './identification';
 
 export default {
   createToken,
-  responseMessage
+  responseMessage,
+  identify
```

Comment:
Find a better and more descriptive name than identify

Reply:
changed